pkg/nsx/services/common/types.go

```diff
@@ -26,7 +36,26 @@ const (
 )
 
 var (
+	ResourceType               string = "resource_type"
 	ResourceTypeSecurityPolicy string = "SecurityPolicy"
 	ResourceTypeGroup          string = "Group"
 	ResourceTypeRule           string = "Rule"
 )
+
+type Service struct {
+	Client          client.Client
+	NSXClient       *nsx.Client
+	NSXConfig       *config.NSXOperatorConfig
+	CacheIndexerMap map[string]cache.Indexer // Cache for all resources by resource type
```

Comment:
if going short var, how about resourceCacheMap, is it better?

Reply:
good